ArduCopter/APM_Config.h

```diff
@@ -39,6 +39,7 @@
 // features below are disabled by default on all boards
 //#define SPRAYER               ENABLED             // enable the crop sprayer feature (two ESC controlled pumps the speed of which depends upon the vehicle's horizontal velocity)
 //#define PRECISION_LANDING     ENABLED             // enable precision landing using companion computer or IRLock sensor
+//#define SOLO_BUTTONS          ENABLED
```

Comment:
Maybe there should just an ifdef called `SOLO_FEATURES`.  There will probably be many more solo things that we'll want to hide behind ifdefs.